src/streamlink_cli/argparser.py

```diff
@@ -1064,6 +1076,17 @@ def build_parser():
         Example: "aac"
         """
     )
+    transport.add_argument(
+        "--ffmpeg-start-at-zero",
+        type=boolean,
+        metavar="STARTATZERO",
+        help="""
+        When used with ffmpeg and copyts,
+        shift input timestamps so they start at zero.
+
+        Default is "True".
+        """
+    )
```

Comment:
The default is `True`. So this should be changed to disable start at zero, ie. `--fffmpeg-no-start-at-zero`.
```suggestion
    transport.add_argument(
        "--ffmpeg-no-start-at-zero",
        action="store_true",
        help="""
        Disables the -start_at_zero ffmpeg option
        when using copyts. 
        """
    )
```